### PR TITLE
refactor: replace enzyme on RNTL

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves []

### Description
[Notion card](https://www.notion.so/artsy/remove-enzyme-from-eigen-b4efdc0fe1cd4f0682d3b4df9c73f628)

### Refactored tests
* Timer
* CountdownTimer
* DurationProvider
* StateManager
* Ticker
* CommercialPartnerInformation
* ContextCard
* ImageCarouselEmbedded
* OtherWorks
* useSpringValue
* TimeOffsetProvider

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Components/Countdown/CountdownTimer.tests.tsx [hunk 1]
```diff
@@ -35,44 +33,41 @@ describe("CountdownTimer", () => {
...
+      <CountdownTimer
+        formattedOpeningHours="Closed"
+        startAt={dateString(Date.now() - 2000)}
+        endAt={dateString(Date.now() - 1000)}
+        countdownComponent={CountdownText}
+      />
     )
-    expect(comp.text()).toContain("")
+
+    expect(getByText("Closed")).toBeTruthy()
```
Comment (gkartalis) [inline src/app/Components/Countdown/CountdownTimer.tests.tsx:71]: 👏

File: src/app/Components/Bidding/Context/TimeOffsetProvider.tests.tsx [hunk 2]
```diff
@@ -1,53 +1,56 @@
...
-  RelayRuntime.fetchQuery.mockReturnValueOnce({
-    toPromise: () =>
-      Promise.resolve({
-        system: {
-          time: {
-            unix: (dateNow - 10 * MINUTES) * 1e-3,
-          },
+  beforeEach(() => {
+    mockEnvironment.mockClear()
+    jest.useFakeTimers()
```
Comment (gkartalis) [inline src/app/Components/Bidding/Context/TimeOffsetProvider.tests.tsx:38]: **suggestion(non-blocking):** minor but we can remove this, don't see it being used anywhere
```suggestion
```

**Review by gkartalis — APPROVED** [review]
🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀

File: src/app/Components/Bidding/Context/TimeOffsetProvider.tests.tsx (hunk 2 above)
Comment (dimatretyak) [reply src/app/Components/Bidding/Context/TimeOffsetProvider.tests.tsx:38]: yeah, make sense 👌

**Review by gkartalis — APPROVED** [review]

**Pull request merged**